QKSMS/src/main/java/com/moez/QKSMS/ui/view/AvatarView.java

```diff
@@ -185,9 +189,10 @@ public void setContactName(String name) {
     }
 
     private boolean isPhoneNumberFormat(String name) {
-        if (TextUtils.isEmpty(name)) {
-            return false;
-        }
+//        We already checked it in first "if" condition in "setContactName"
+//        if (TextUtils.isEmpty(name)) {
+//            return false;
+//        }
```

Comment:
It's still good to leave this check here, because a different string could technically be passed in, and then it would cause the app to crash in that case